examples/posthumous-events/index.mjs

```diff
@@ -0,0 +1,17 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+const stdlib = loadStdlib(process.env);
+
+const startingBalance = stdlib.parseCurrency(100);
+const [ accAlice ] = await stdlib.newTestAccounts(1, startingBalance);
+const ctc = accAlice.contract(backend);
+
+// After this Promise finishes, the contract has exited
+await backend.Alice(ctc, {});
+
+// 'Posthumous' code, looking at events emitted while the contract was alive
+const numMonPromise = 
+  ctc.events.Numbers.number.monitor(ev => console.log(`Number event #${ev.what}`));
```

Comment:
This is might not be a good test because `ctc` has the data from the initial run. You should make a NEW contract object with a new account based on `ctc.getInfo()` after Alice is over